### ISSUE TITLE
Dont know what this error is but my server reported it

### ISSUE THREAD

**Issue by Squawkykaka** [dmulloy2/ProtocolLib#2844]
- [ ] This issue is not solved in a development build

**Describe the bug**
  Stack Trace:
  java.lang.IllegalArgumentException: Unable to find a field that matches {modifiers=[required: 10000, banned: 1000], type={ type input instanceof interface java.util.Map }}
        at ProtocolLib.jar//com.comphenix.protocol.reflect.FuzzyReflection.getField(FuzzyReflection.java:352)
        at ProtocolLib.jar//com.comphenix.protocol.injector.packet.PacketRegistry.createNewRegister(PacketRegistry.java:188)
        at ProtocolLib.jar//com.comphenix.protocol.injector.packet.PacketRegistry.initialize(PacketRegistry.java:292)
        at ProtocolLib.jar//com.comphenix.protocol.injector.packet.PacketRegistry.getClientPacketTypes(PacketRegistry.java:327)
        at ProtocolLib.jar//com.comphenix.protocol.injector.PacketFilterManager.<init>(PacketFilterManager.java:120)
        at ProtocolLib.jar//com.comphenix.protocol.injector.PacketFilterBuilder.build(PacketFilterBuilder.java:121)
        at ProtocolLib.jar//com.comphenix.protocol.ProtocolLib.onLoad(ProtocolLib.java:183)
        at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:59)
        at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:18)
        at io.papermc.paper.plugin.storage.SimpleProviderStorage.enter(SimpleProviderStorage.java:39)
        at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enter(LaunchEntryPointHandler.java:36)
        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.loadPlugins(CraftServer.java:519)
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:310)
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1148)
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:326)
        at java.base/java.lang.Thread.run(Thread.java:840)
  Dump:
  Parameters: 
    [NULL]
  Sender:
    com.comphenix.protocol.ProtocolLib@119b537[
      statistics=<null>
      packetTask=<null>
      tickCounter=0
      configExpectedMod=-1
      updater=com.comphenix.protocol.updater.SpigotUpdater@81b2128
      redirectHandler=<null>
      scheduler=com.comphenix.protocol.scheduler.DefaultScheduler@5f5f7507
      commandProtocol=<null>
      commandPacket=<null>
      commandFilter=<null>
      packetLogging=<null>
      skipDisable=false
      isEnabled=false
      loader=io.papermc.paper.plugin.manager.DummyBukkitPluginLoader@56d96a3c
      server=CraftServer{serverName=Purpur,serverVersion=git-Purpur-2124,minecraftVersion=1.20.4}
      file=plugins/ProtocolLib.jar
      description=org.bukkit.plugin.PluginDescriptionFile@4be6bdc0
      pluginMeta=org.bukkit.plugin.PluginDescriptionFile@4be6bdc0
      dataFolder=plugins/ProtocolLib
      classLoader=PluginClassLoader{plugin=ProtocolLib v5.1.0, pluginEnabled=false, url=plugins/ProtocolLib.jar}
      naggable=true
      newConfig=YamlConfiguration[path='', root='YamlConfiguration']
      configFile=plugins/ProtocolLib/config.yml
      logger=com.destroystokyo.paper.utils.PaperPluginLogger@722011b6
    ]
  Version:
    ProtocolLib v5.1.0
  Java Version:
    17.0.10
  Server:
    git-Purpur-2124 (MC: 1.20.4)

**To Reproduce**

**Expected behavior**
ProtocolLib starting it crashes at the moment

[20:45:38 INFO]: Paper Plugins: (1):
[20:45:38 INFO]:  - HuskHomes
[20:45:38 INFO]: Bukkit Plugins: (18):
[20:45:38 INFO]:  - *AdminInventory, Chunky, CoreProtect, FastAsyncWorldEdit, GreetingsMessages, LuckPerms, Maintenance, Plan, ProtocolLib (Broken), RHLeafDecay
[20:45:38 INFO]:  ServerBackup, SkinsRestorer, spark, SuperVanish, TabTPS, Vault, voicechat, WorldGuard

**Comment by HopeJ-Personal** [comment]
I have virtually the same report, still no clue but for me atleast its 1.20.2 on version 5.1.0 with a paper server (paper version 318 for 1.20.2), heres my log

~~~
[16:59:46 INFO]: Default game type: SURVIVAL
[16:59:46 INFO]: Generating keypair
[16:59:46 INFO]: Starting Minecraft server on *:25565
[16:59:46 INFO]: Using default channel type
[16:59:46 INFO]: Paper: Using Java compression from Velocity.
[16:59:46 INFO]: Paper: Using Java cipher from Velocity.
[16:59:46 INFO]: [ProtocolLib] Loading server plugin ProtocolLib v5.1.0
[16:59:47 WARN]: [ProtocolLib] Version (MC: 1.20.2) has not yet been tested! Proceed with caution.
[16:59:47 ERROR]:   [ProtocolLib] INTERNAL ERROR: Cannot load ProtocolLib.
  If this problem hasn't already been reported, please open a ticket
  at https://github.com/dmulloy2/ProtocolLib/issues with the following data:
  Stack Trace:
  java.lang.IllegalArgumentException: Unable to find a field that matches {modifiers=[required: 10000, banned: 1000], type={ type input instanceof interface java.util.Map }}
        at ProtocolLib.jar//com.comphenix.protocol.reflect.FuzzyReflection.getField(FuzzyReflection.java:352)
        at ProtocolLib.jar//com.comphenix.protocol.injector.packet.PacketRegistry.createNewRegister(PacketRegistry.java:188)
        at ProtocolLib.jar//com.comphenix.protocol.injector.packet.PacketRegistry.initialize(PacketRegistry.java:292)
        at ProtocolLib.jar//com.comphenix.protocol.injector.packet.PacketRegistry.getClientPacketTypes(PacketRegistry.java:327)
        at ProtocolLib.jar//com.comphenix.protocol.injector.PacketFilterManager.<init>(PacketFilterManager.java:120)
        at ProtocolLib.jar//com.comphenix.protocol.injector.PacketFilterBuilder.build(PacketFilterBuilder.java:121)
        at ProtocolLib.jar//com.comphenix.protocol.ProtocolLib.onLoad(ProtocolLib.java:183)
        at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:59)
        at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:18)
        at io.papermc.paper.plugin.storage.SimpleProviderStorage.enter(SimpleProviderStorage.java:39)
        at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enter(LaunchEntryPointHandler.java:36)
        at org.bukkit.craftbukkit.v1_20_R2.CraftServer.loadPlugins(CraftServer.java:514)
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:273)
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1086)
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315)
        at java.base/java.lang.Thread.run(Thread.java:1583)
  Dump:
  Parameters:
    [NULL]
  Sender:
    com.comphenix.protocol.ProtocolLib@5b6465c0[
      statistics=<null>
      packetTask=<null>
      tickCounter=0
      configExpectedMod=-1
      updater=com.comphenix.protocol.updater.SpigotUpdater@32e64067
      redirectHandler=<null>
      scheduler=com.comphenix.protocol.scheduler.DefaultScheduler@5d17d8c8
      commandProtocol=<null>
      commandPacket=<null>
      commandFilter=<null>
      packetLogging=<null>
      skipDisable=false
      isEnabled=false
      loader=io.papermc.paper.plugin.manager.DummyBukkitPluginLoader@aa2f917
      server=CraftServer{serverName=Paper,serverVersion=git-Paper-318,minecraftVersion=1.20.2}
      file=plugins\ProtocolLib.jar
      description=org.bukkit.plugin.PluginDescriptionFile@6fd250b1
      pluginMeta=org.bukkit.plugin.PluginDescriptionFile@6fd250b1
      dataFolder=plugins\ProtocolLib
      classLoader=PluginClassLoader{plugin=ProtocolLib v5.1.0, pluginEnabled=false, url=plugins\ProtocolLib.jar}
      naggable=true
      newConfig=YamlConfiguration[path='', root='YamlConfiguration']
      configFile=plugins\ProtocolLib\config.yml
      logger=com.destroystokyo.paper.utils.PaperPluginLogger@318b5478
    ]
  Version:
    ProtocolLib v5.1.0
  Java Version:
    21.0.1
  Server:
    git-Paper-318 (MC: 1.20.2)

[16:59:47 INFO]: Server permissions file permissions.yml is empty, ignoring it
[16:59:47 INFO]: [ProtocolLib] Enabling ProtocolLib v5.1.0
[16:59:47 ERROR]: ******************************************************
[16:59:47 ERROR]: *** ProtocolLib does not support plugin reloaders! ***
[16:59:47 ERROR]: *** Please use the built-in reload command!        ***
[16:59:47 ERROR]: ******************************************************
[16:59:47 INFO]: [ProtocolLib] Disabling ProtocolLib v5.1.0
~~~

**Comment by Squawkykaka** [comment]
its fixed on the latest dev build for 1.20.4
